Modules/booking/Controller/BookingdefaultController.php

```diff
@@ -831,75 +767,36 @@ public function deleteperiodAction($id_space, $id_period){
-        Configuration::getLogger()->debug("bookingdefaultController", ["email?", $sendmail]);
-        if ($sendmail == 1) {
-            // get the resource
-            $modelCalEntry = new BkCalendarEntry();
-            $entryInfo = $modelCalEntry->getEntry($id);
-            $id_resource = $entryInfo["resource_id"];
-
-
-            $resourceModel = new ResourceInfo();
-            $resourceName = $resourceModel->getName($id_resource);
-
-            // get the space name
-            $modelSpace = new CoreSpace();
-            $space = $modelSpace->getSpace($id_space);
-
-            // user info
-            // $userModel = new CoreUser();
-            // $user = $userModel->getInfo($_SESSION["id_user"]);
-
-            // mail content
-            //$from = $user["email"];
-            $from = Configuration::get('smtp_from');
-            $fromName = "Platform-Manager";
-
-            $toAdress = $modelCalEntry->getEmailsBookerResource($id_resource);
-            $subject = "[" . $space["name"] . "]" .$resourceName . " has been freed";
-            $content = "The " . $resourceName . " has been freed from " . date("Y-m-d H:i", $entryInfo["start_time"]) . " to " . date("Y-m-d H:i", $entryInfo["end_time"]);
-
-            //echo "send email: from " . $from . ", subject " . $subject . ", content: " . $content;
-            // send the email
-            $mailerModel = new MailerSend();
-            $mailerModel->sendEmail($from, $fromName, $toAdress, $subject, $content);
-        }
-
-        $modelConfig = new CoreConfig();
-        $sendMailResponsibles = $modelConfig->getParamSpace("BkBookingMailingAdmins", $id_space);
-        if ( $sendMailResponsibles > 0){
-
-            // get the resource
+        $sendEmail = $this->request->getParameter("sendmail");
+        if ($sendEmail == 1) {
             $modelCalEntry = new BkCalendarEntry();
             $entryInfo = $modelCalEntry->getEntry($id);
             $id_resource = $entryInfo["resource_id"];
-
             $resourceModel = new ResourceInfo();
             $resourceName = $resourceModel->getName($id_resource);
 
-            // user info
-            // $userModel = new CoreUser();
-            // $user = $userModel->getInfo($_SESSION["id_user"]);
-
-            // get the resource responsibles
-            $modelResource = new ReResps();
-
-            // mail content
-            // $from = $user["email"];
-            $from = Configuration::get('smtp_from');
-            $fromName = "Platform-Manager";
+            $toAddress = $modelCalEntry->getEmailsBookerResource($id_resource);
 
-            $toAdress = $modelResource->getResourcesManagersEmails($id_resource);
-            $subject = "[". $space["name"] ."]" . $resourceName . " has been freed";
+            $modelConfig = new CoreConfig();
+            $sendMailResponsibles = $modelConfig->getParamSpace("BkBookingMailingAdmins", $id_space);
+            
+            if ($sendMailResponsibles > 0) {
```

Comment:
php8 comparaison string/int again, works?

Reply:
sill works. I gonna try to explicitly make the conversions when I see that